docs/rules/no-restricted-syntax.md

```diff
@@ -1,9 +1,11 @@
 # disallow specified syntax (no-restricted-syntax)
 
-JavaScript has a lot of language features, and not everyone likes all of them. As a result, some projects choose to disallow the use of certain language features altogether. For instance, you might decide to disallow the use of `try-catch` or `class`.
+JavaScript has a lot of language features, and not everyone likes all of them. As a result, some projects choose to disallow the use of certain language features altogether. For instance, you might decide to disallow the use of `try-catch` or `class`, or you might decide to disallow the use of the `in` operator.
```

Comment:
The change on this line is cool, but I don't see any examples showing how to do that. Conversely, the examples show how to avoid using `require()` calls. Could you please resolve this inconsistency (ideally by adding an example for forbidding the `in` operator)?